app/components/publications/publication/proofs/proof-info-panel.js

```diff
@@ -0,0 +1,55 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+/**
+ * @argument {PublicationSubcase}
+ * @argument {PublicationFlow}
+ * @argument {onSave}
+ */
+export default class PublicationsPublicationProofsProofInfoPanelComponent extends Component {
+  @service store;
+
+  @tracked isEditing = false;
+  @tracked proofPrintCorrector;
+  @tracked publicationStatus;
+
+  @tracked dueDate;
+
+  constructor() {
+    super(...arguments);
+    this.loadStatus.perform();
+  }
+
+  @task
+  *loadStatus() {
+    this.publicationStatus = yield this.args.publicationFlow.status;
+  }
+
+  @action
+  openEditingPanel() {
+    this.isEditing = true;
+    this.proofPrintCorrector = this.args.publicationSubcase.proofPrintCorrector;
+    this.dueDate = this.args.publicationSubcase.dueDate;
```

Comment:
Áls de save volledig uitgevoerd wordt in de component, is deze extra layer niet nodig en kan er rechtstreeks op de attributes van de publication-subcase gewerkt worden (met een rollback dan bij het sluiten van het edit-panel zonder opslaan)

Reply:
Deze is toch niet resolved? Je werkt toch nog steeds op een copy van de attributes i.p.v. rechtstreeks op `@publicationSubcase`? Dat kan de component nog fel vereenvoudigen.